thesis-gui/src/components/my-quizzes/MyQuizzes.tsx

```diff
@@ -1,14 +1,14 @@
 import React, {useEffect, useState} from "react";
 import {
-    IonButton,
+    IonButton, IonButtons,
     IonContent, IonIcon,
     IonItem,
     IonLabel,
     IonList,
     IonPage, IonText, useIonToast
 } from "@ionic/react";
 import {useHistory} from "react-router";
-import {keyOutline} from "ionicons/icons";
+import {keyOutline, create, trashBin} from "ionicons/icons";
```

Comment:
I would use the outlined icons ;-) For me looks better but it's only my opinion ;-)

Reply:
Why not :) Done.